docs/vwii/vwii-modding.md

```diff
@@ -16,16 +16,16 @@ We will now place the required Homebrew files on the SD Card.
 - The latest files from [Tiramisu for your café](https://tiramisu.foryour.cafe).
     - Select `Nanddumper` and `Base-Tiramisu`.

```

Comment:
I'm unsure if this line is needed? When I go to download Tiramisu from the link, I'm not shown any choices to select.